2022-01-11 11:31:21,892 - INFO basePage.py [line:88]: 查找"输入账号"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:21,892 - INFO basePage.py [line:88]: 查找"输入账号"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:21,904 - INFO basePage.py [line:113]: 在"输入账号"输入"pengjie",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:21,904 - INFO basePage.py [line:113]: 在"输入账号"输入"pengjie",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:21,975 - INFO basePage.py [line:88]: 查找"输入密码"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:21,975 - INFO basePage.py [line:88]: 查找"输入密码"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:21,987 - INFO basePage.py [line:113]: 在"输入密码"输入"qq123",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:21,987 - INFO basePage.py [line:113]: 在"输入密码"输入"qq123",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:22,040 - INFO basePage.py [line:88]: 查找"点击登录"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:22,040 - INFO basePage.py [line:88]: 查找"点击登录"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:22,051 - INFO basePage.py [line:140]: 点击"点击登录",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:22,051 - INFO basePage.py [line:140]: 点击"点击登录",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:24,094 - INFO basePage.py [line:54]: 等待"断言"消失,元素定位:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[3]/div[2]/span/span/button/span")
2022-01-11 11:31:24,094 - INFO basePage.py [line:54]: 等待"断言"消失,元素定位:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[3]/div[2]/span/span/button/span")
2022-01-11 11:31:34,141 - INFO basePage.py [line:59]: 等待"断言"时长:0:00:10.044252
2022-01-11 11:31:34,141 - INFO basePage.py [line:59]: 等待"断言"时长:0:00:10.044252
2022-01-11 11:31:52,517 - INFO basePage.py [line:88]: 查找"输入账号"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:52,517 - INFO basePage.py [line:88]: 查找"输入账号"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:52,530 - INFO basePage.py [line:113]: 在"输入账号"输入"oms1212",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:52,530 - INFO basePage.py [line:113]: 在"输入账号"输入"oms1212",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:31:52,601 - INFO basePage.py [line:88]: 查找"输入密码"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:52,601 - INFO basePage.py [line:88]: 查找"输入密码"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:52,614 - INFO basePage.py [line:113]: 在"输入密码"输入"Qq123456",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:52,614 - INFO basePage.py [line:113]: 在"输入密码"输入"Qq123456",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:31:52,674 - INFO basePage.py [line:88]: 查找"点击登录"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:52,674 - INFO basePage.py [line:88]: 查找"点击登录"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:52,684 - INFO basePage.py [line:140]: 点击"点击登录",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:52,684 - INFO basePage.py [line:140]: 点击"点击登录",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:31:54,733 - INFO basePage.py [line:88]: 查找"供应链管理"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:31:54,733 - INFO basePage.py [line:88]: 查找"供应链管理"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:31:54,771 - INFO basePage.py [line:140]: 点击"供应链管理",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:31:54,771 - INFO basePage.py [line:140]: 点击"供应链管理",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:32:04,822 - ERROR basePage.py [line:144]: "供应链管理"点击失败
Traceback (most recent call last):
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:04,822 - ERROR basePage.py [line:144]: "供应链管理"点击失败
Traceback (most recent call last):
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:04,917 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\[供应链管理]点击异常_20220111_113204.png
2022-01-11 11:32:04,917 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\[供应链管理]点击异常_20220111_113204.png
2022-01-11 11:32:04,918 - INFO basePage.py [line:88]: 查找"展开模块"元素，元素定位:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
2022-01-11 11:32:04,918 - INFO basePage.py [line:88]: 查找"展开模块"元素，元素定位:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
2022-01-11 11:32:14,949 - ERROR basePage.py [line:92]: 查找"展开模块"元素失败,定位方式:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
Traceback (most recent call last):
  File "..\PageObject\omsIndex.py", line 36, in OMS_p
    self.click_element(self.A1,model="供应链管理")
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "..\Common\basePage.py", line 90, in find_element
    return self.driver.find_element(*loc)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path"}
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:14,949 - ERROR basePage.py [line:92]: 查找"展开模块"元素失败,定位方式:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
Traceback (most recent call last):
  File "..\PageObject\omsIndex.py", line 36, in OMS_p
    self.click_element(self.A1,model="供应链管理")
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "..\Common\basePage.py", line 90, in find_element
    return self.driver.find_element(*loc)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path"}
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:15,017 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\查找元素[展开模块]异常_20220111_113214.png
2022-01-11 11:32:15,017 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\查找元素[展开模块]异常_20220111_113214.png
2022-01-11 11:32:33,552 - INFO basePage.py [line:88]: 查找"输入账号"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:32:33,552 - INFO basePage.py [line:88]: 查找"输入账号"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:32:33,565 - INFO basePage.py [line:113]: 在"输入账号"输入"oms1212",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:32:33,565 - INFO basePage.py [line:113]: 在"输入账号"输入"oms1212",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[2]/div/div/input")
2022-01-11 11:32:33,638 - INFO basePage.py [line:88]: 查找"输入密码"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:32:33,638 - INFO basePage.py [line:88]: 查找"输入密码"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:32:33,650 - INFO basePage.py [line:113]: 在"输入密码"输入"Qq123456",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:32:33,650 - INFO basePage.py [line:113]: 在"输入密码"输入"Qq123456",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/div[3]/div/div/input")
2022-01-11 11:32:33,712 - INFO basePage.py [line:88]: 查找"点击登录"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:32:33,712 - INFO basePage.py [line:88]: 查找"点击登录"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:32:33,722 - INFO basePage.py [line:140]: 点击"点击登录",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:32:33,722 - INFO basePage.py [line:140]: 点击"点击登录",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/form/button")
2022-01-11 11:32:35,770 - INFO basePage.py [line:88]: 查找"供应链管理"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:32:35,770 - INFO basePage.py [line:88]: 查找"供应链管理"元素，元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:32:35,787 - INFO basePage.py [line:140]: 点击"供应链管理",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:32:35,787 - INFO basePage.py [line:140]: 点击"供应链管理",元素定位:('xpath', "//*[@id='app']/div/div[1]/div[2]/div[1]/div/ul/div[2]/li/div/span")
2022-01-11 11:32:45,809 - ERROR basePage.py [line:144]: "供应链管理"点击失败
Traceback (most recent call last):
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:45,809 - ERROR basePage.py [line:144]: "供应链管理"点击失败
Traceback (most recent call last):
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:45,884 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\[供应链管理]点击异常_20220111_113245.png
2022-01-11 11:32:45,884 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\[供应链管理]点击异常_20220111_113245.png
2022-01-11 11:32:45,885 - INFO basePage.py [line:88]: 查找"展开模块"元素，元素定位:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
2022-01-11 11:32:45,885 - INFO basePage.py [line:88]: 查找"展开模块"元素，元素定位:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
2022-01-11 11:32:55,943 - ERROR basePage.py [line:92]: 查找"展开模块"元素失败,定位方式:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
Traceback (most recent call last):
  File "..\PageObject\omsIndex.py", line 36, in OMS_p
    self.click_element(self.A1,model="供应链管理")
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "..\Common\basePage.py", line 90, in find_element
    return self.driver.find_element(*loc)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path"}
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:55,943 - ERROR basePage.py [line:92]: 查找"展开模块"元素失败,定位方式:('xpath', "//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path")
Traceback (most recent call last):
  File "..\PageObject\omsIndex.py", line 36, in OMS_p
    self.click_element(self.A1,model="供应链管理")
  File "..\Common\basePage.py", line 142, in click_element
    ele.click()
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=96.0.4664.110)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "..\Common\basePage.py", line 90, in find_element
    return self.driver.find_element(*loc)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\bonjour\appdata\local\programs\python\python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='app']/div/div[2]/div/div[1]/div[1]/svg/path"}
  (Session info: headless chrome=96.0.4664.110)

2022-01-11 11:32:55,991 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\查找元素[展开模块]异常_20220111_113255.png
2022-01-11 11:32:55,991 - INFO basePage.py [line:247]: 截屏成功,图片路径为E:\12\Screenshots\2022-01-11\查找元素[展开模块]异常_20220111_113255.png
